test/transforms.jl

```diff
@@ -249,14 +252,15 @@
       p₂ = scatter(n.x, n.y, label="Scale")
       p = plot(p₁, p₂, layout=(1,2))
 
-      @test_reference joinpath(datadir,"scale.png") p
+      @test_reference joinpath(datadir, "scale.png") p
+      # savefig(p, joinpath(datadir, "scale.png"))
```

Comment:
```suggestion
```

Reply:
I removed the comments of savefig function.